### PR TITLE
Dashboard page front-end

### PR DESCRIPTION
**Pull request by ichala**
Create the dashboard page that contains some stats and charts about all the databases data

### CODE REVIEW

File: src/Pages/Admin/Dashboard/Tables/UserStatsTable.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+              </tr>
+            </thead>
+            <tbody>
+              {stats.map((user) => (
+                <tr key={user.email}>
+                  <td>
+                    {user.name}
+                    {' '}
+                    {user.last_name}
+                  </td>
```
Comment (aimalamiri) [inline src/Pages/Admin/Dashboard/Tables/UserStatsTable.jsx:26]: [Optional] So you want to put a `space ` in here, but it looks quite long, here is a simple approach 
```javascript
<td>{`${user.name} ${user.last_name}`}</td>
 ````

What do you think?

**Review by aimalamiri — APPROVED** [review]
### **Hi 👋** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: src/Pages/Admin/Dashboard/Tables/UserStatsTable.jsx (hunk 1 above)
Comment (ichala) [reply src/Pages/Admin/Dashboard/Tables/UserStatsTable.jsx:26]: @aimalamiri yeah true , but i didn't even put that , thats generated by stylelint 😅 ill try to remove it

**Review by EidHachem — APPROVED** [review]
Hi, @ichala , Great Job Implementing the requirements.
You can go ahead and merge the project now

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6433317732347073475965656b435a792f67697068792e676966

## Highlights
✅ App has the correct functionality 🔥 
✅ Good commit messages 👍 

Cheers and Happy coding!👏👏👏

**Review by d2ndjim — APPROVED** [review]
Hey,
@ichala 
approve

PR Review result:  APPROVED! 👍 👍 👍 

Good job so far. 

- No linter errors
- Well structured code base

It's time to merge 😃

**Pull request merged**